src/plcc.py

```diff
@@ -668,12 +690,44 @@ def makeAbstractStub(base):
         }}
     }}
 
-//{base}//
+    //{base}//
 }}
 """.format(cls=cls,
            base=base,
            ext=ext,
-           cases='\n'.join(indent(2, caseList))
+           cases='\n\t\t\t'.join(caseList)
```

Comment:
Why not use indent() as before? I'm suspicious of hardcoding tabs here when there was a nice function that added the indents before.